autogen/safer-cluster/variables.tf.tmpl

```diff
@@ -202,6 +202,28 @@ variable "node_pools_oauth_scopes" {
   }
 }
 
+variable "cluster_autoscaling" {
+  type = object({
+    enabled             = bool
+    autoscaling_profile = string
+    min_cpu_cores       = number
+    max_cpu_cores       = number
+    min_memory_gb       = number
+    max_memory_gb       = number
+    gpu_resources       = list(object({ resource_type = string, minimum = number, maximum = number }))
+  })
+  default = {
```

Comment:
I don't love that this is duplicated from the underlying beta cluster, but making it nullable and defaulting to null feels like it hides defaults too much

Reply:
Yeah we may look at rearchitecting some of these objects with optionals in the future now that 1.3 supports it.